sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -66,12 +66,29 @@ struct sof_card_private {
 	bool common_hdmi_codec_drv;
 };
 
+const char *sof_override_tplg_name;
+
 static int sof_rt5682_quirk_cb(const struct dmi_system_id *id)
 {
 	sof_rt5682_quirk = (unsigned long)id->driver_data;
 	return 1;
 }
-
```

Comment:
please, keep empty lines between functions

Reply:
> please, keep empty lines between functions

Done now.

Reply:
sorry, I also meant this empty line, looks like you're still removing it?

Reply:
Thank you, corrected it now.